test/TensorFlow/integration.swift

```diff
@@ -228,6 +228,65 @@ public func test_bool_param2(cond: Bool, // expected-warning {{'cond' implicitly
 // CHECK-NEXT: [[PROGRAM:%.*]] = apply [[STARTFN:%.*]](
 // CHECK: cond_br [[BOOLVAL]],
 
+public func test_multiple_ifs(status: Bool // expected-warning {{'status' implicitly copied to the accelerator}}
+) {
+  var a = Tensor<Int32>(0);
+  let b = a;
+  if (status) { // expected-note {{value used here}}
+    a += b;
+  }
+  a += b;
+  if (status) {
+    a += b;
+  }
+  a -= b;
+  print (a)
+}
+
+// CHECK-LABEL: --- TFPartition Accelerator Result: {{.*}}test_multiple_ifs{{.*}}
```

Comment:
Does CHECKing "TFPartition Accelerator Result" in this test add value? It seems CHECKing on "XLA CFG Canonicalize" would be sufficient. If we must, maybe also check on the presence of "cond_br".

Minimizing the CHECK'ed content would help reduce test maintenance efforts.

e.g. when we later add a new attr to builtin "__tfop_Const", or migrate it to the new graph op inst, or rename "tf_tensor_to_i1" to something else, etc.

Reply:
Good point. I simplified the test. I will also move this over later to a separate file that checks canonicalization and SESE region construction.